Run failed after update (eg: cause 502) / アップデートしたら502とかになる

## 💡 Summary
When you update Misskey, the build succeeds but may fail to start.
(In most production environments, 502 occurs.)
Run `yarn --force` to fix it.

Misskeyをupdateすると、ビルドは成功するものの起動に失敗することがある。
(大抵のproduction環境では502が発生する)
`yarn --force` でなおる。

※Misskeyのバージョン問わず起きる
※たぶん失敗する時は、 node_modules がデフォルトに戻っているにも関わらず`Building fresh packages...`でsharpのビルドがトリガされていない

<!-- Tell us what the bug is -->
## 🙂 Expected Behavior
起動に成功する

<!--- Tell us what should happen -->
## ☹️ Actual Behavior
起動に失敗する

<!--- Tell us what happens instead of the expected behavior -->

## 📝 Steps to Reproduce
一度ビルド後に違うバージョンをcheckoutする
例えば 12.53.0 -> 12.55.0, 12.55.0 -> 12.53.0

```
yarn && yarn build && yarn start 
```
```
Error: 
Something went wrong installing the "sharp" module

Cannot find module '../build/Release/sharp.node'
Require stack:
- MISSKEY_PATH/node_modules/sharp/lib/constructor.js
- MISSKEY_PATH/node_modules/sharp/lib/index.js
- MISSKEY_PATH/built/services/drive/image-processor.js
- MISSKEY_PATH/built/services/drive/generate-video-thumbnail.js
- MISSKEY_PATH/built/services/drive/add-file.js
- MISSKEY_PATH/built/services/drive/upload-from-url.js
- MISSKEY_PATH/built/remote/activitypub/models/image.js
- MISSKEY_PATH/built/remote/activitypub/models/person.js
- MISSKEY_PATH/built/remote/resolve-user.js
- MISSKEY_PATH/built/services/note/create.js
- MISSKEY_PATH/built/remote/activitypub/models/note.js
- MISSKEY_PATH/built/remote/activitypub/kernel/create/note.js
- MISSKEY_PATH/built/remote/activitypub/kernel/create/index.js
- MISSKEY_PATH/built/remote/activitypub/kernel/index.js
- MISSKEY_PATH/built/remote/activitypub/perform.js
- MISSKEY_PATH/built/queue/processors/inbox.js
- MISSKEY_PATH/built/queue/index.js
- MISSKEY_PATH/built/server/activitypub.js
- MISSKEY_PATH/built/server/index.js
- MISSKEY_PATH/built/boot/worker.js
- MISSKEY_PATH/built/boot/index.js
- MISSKEY_PATH/built/index.js
- MISSKEY_PATH/index.js

- Run "npm rebuild --verbose sharp" and look for errors
- Consult the installation documentation at https://sharp.pixelplumbing.com/install
- Search for this error at https://github.com/lovell/sharp/issues
```

※sharpでなくre2で出ることもある https://github.com/syuilo/misskey/issues/6829

## 📌 Environment
node: v14.4.0
yarn: v1.22.10
npm: v6.14.5
<!-- Tell us where on the platform it happens -->

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -43,6 +43,7 @@ You should also include the user name that made the change.
 - Server: use correct order of attachments on notes @Johann150
 - Server: prevent crash when processing certain PNGs @syuilo
 - Server: Fix unable to generate video thumbnails @mei23
+- Server: Fix `Cannot find module` issue @mei23
 
 ## 12.110.1 (2022/04/23)
 

--- a/scripts/install-packages.js
+++ b/scripts/install-packages.js
@@ -3,7 +3,7 @@ const execa = require('execa');
 (async () => {
 	console.log('installing dependencies of packages/backend ...');
 
-	await execa('yarn', ['install'], {
+	await execa('yarn', ['--force', 'install'], {
 		cwd: __dirname + '/../packages/backend',
 		stdout: process.stdout,
 		stderr: process.stderr,

